src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -359,6 +359,109 @@ public static String csvToHtmlTable(String str) {
                              result.toString());
     }
 
+    private static List<String> combineRowDataEntries(List<String> rowData,
+            String delimiter) {
```

Comment:
Given the function name, i might expect it to combine all elements in the list using the delimiter and return a string, but it is on;y combining the first 2 String elements and returning a List? The function needs to be documented/renamed.